plutus-core/plutus-ir/test/PlutusIR/Compiler/Recursion/factorial.golden

```diff
@@ -2,38 +2,38 @@
   1.1.0
   [
     [
-      (lam s_800 [ s_800 s_800 ])
+      (lam s_836 [ s_836 s_836 ])
```

Comment:
Kinda weird that only uniques change in this file and nothing else. CSE performs an additional renaming?